[DNS] Exposed DNS settings in dash (#13149)
cloudflare/cloudflare-docs: content/dns/dnssec/multi-signer-dnssec/setup.md

## 1. Set up Cloudflare zone

### Cloudflare as Primary (full setup)

If you use Cloudflare as a primary DNS provider, meaning that you manage your DNS records in Cloudflare, do the following:

{{<tabs labels="Dashboard | API">}}
{{<tab label="dashboard" no-code="true">}}

> 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account and zone.

Reviewer: ```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account and zone.
```
Nit: Some of the dashboard URLs in this PR have `/login` and others don't -- I usually don't include any path.

--header "X-Auth-Email: <EMAIL>" \
--header "X-Auth-Key: <API_KEY>" \
--header "Content-Type: application/json" \
--data '{
    "multi_provider": true
}'
```

{{</tab>}}
{{</tabs>}}

### Cloudflare as Secondary

If you use Cloudflare as a secondary DNS provider, do the following:

{{<tabs labels="Dashboard | API">}}
{{<tab label="dashboard" no-code="true">}}

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account and zone.
2. Go to **DNS** > **Settings**.
3. For **DNSSEC with Secondary DNS** select **Live signing**.

{{<Aside type="note">}}
For the purpose of this tutorial, you will update your registrar with the DS record later, in [Step 3](/dns/dnssec/multi-signer-dnssec/setup/#3-set-up-registrar).
{{</Aside>}}

5. Also enable **Multi-signer DNSSEC**.
6. Add the ZSK(s) of your external provider(s) to a DNSKEY record at your primary DNS provider. This record should be transferred successfully to Cloudflare.
7. Add your external provider(s) nameservers as NS records on your zone apex at your primary DNS provider. These records should be transferred successfully to Cloudflare.

{{</tab>}}
{{<tab label="api" no-code="true">}}

> 1. Use the [Edit DNSSEC Status endpoint](/api/operations/dnssec-edit-dnssec-status) to enable DNSSEC and activate multi-signer DNSSEC for your zone. This is done by setting `status` to `active` and `dnssec_multi_signer` to `true`, as in the following example.

Reviewer: ```suggestion
1. Use the [Edit DNSSEC Status endpoint](/api/operations/dnssec-edit-dnssec-status) to enable DNSSEC and activate multi-signer DNSSEC for your zone. Set `status` to `active` and `dnssec_multi_signer` to `true`, as in the following example.
```
